BigML.Tests/TestFusions.cs

```diff
@@ -50,13 +52,13 @@ public async Task CreateFusionFromRemoteSource()
 
             // Create Fusion
             Fusion.Arguments argsFs = new Fusion.Arguments();
-            List<dynamic> modelIDs = new List<dynamic>();
+            System.Collections.Generic.List<dynamic> modelIDs = new System.Collections.Generic.List<dynamic>();
```

Comment:
Why not add "using System.Collections.Generic" (as before), then you can just use List<dynamic>. Also since this list will always store model ids, couldn't you use List<string>?

Reply:
I can add the reference at the top. As to using List instead of the dynamic variable, that's preexisting code, so I'm not sure about the reason but I can think of one. The information about the models can be provided as a list of model ids or also as a list of maps that contain model ID and weight associated to the model. Probably that's the cause for the dynamic typing.